docs/python_docs/python/api/gluon/model_zoo/index.rst

```diff
@@ -15,21 +15,20 @@
    specific language governing permissions and limitations
    under the License.
 
-model_zoo.vision
-=================
+gluon.model_zoo.vision
+======================
 
 .. automodule:: mxnet.gluon.model_zoo.vision
+   :noindex:
```

Comment:
What is the reason on when you use this approach:
```
.. automodule:: mxnet.gluon.model_zoo.vision
   :noindex:
``` 
...versus this:
```
.. automodule:: mxnet.gluon.loss
    :members:
    :imported-members:
    :autosummary:
```

Reply:
The difference is this.

For model_zoo.vision, I wanted to keep the documentation organized by the different model architectures: resnet, vgg etcs. it would have been super cluttered if it was just done all automatically. There are a ton of pretrained models in the model zoo so it's worth it to have them organized by sections.

The second approach in mxnet.gluon.loss automatically generates the documentation for all classes in the package and :autosummary: allows you to create a summary table at top automatically. Since there are not that many loss functions the autogenerated table summary is sufficient for this class.